### PR TITLE
The a flag is available with printf formatting

### PR DESCRIPTION
**Pull request by Akuli**
Currently pyformat.info says that printf-formatting doesn't do `ascii()`.

### CODE REVIEW

File: tests/test_content.py [hunk 1]
```diff
@@ -83,9 +83,11 @@ class Data(object):
         def __repr__(self):
             return 'räpr'
 
+    old_result = '%r %a' % (Data(), Data())
     new_result = '{0!r} {0!a}'.format(Data())
 
     assert new_result == 'räpr r\\xe4pr'  # output
+    assert new_result == old_result
```
Comment (madphysicist) [inline tests/test_content.py]: Would it make more sense to do this:
```
assert old_result == 'r....'
assert new_result == old_result
```

Comment (Akuli) [inline tests/test_content.py]: `assert old_result == new_result == '...'` could be nice also.

Comment (ulope) [inline tests/test_content.py]: The way you did it is "correct". The reason is that we produce the page content from the parsed tests. So there has to be a fixed structure to the examples.

However we're currently working on a [version 2](https://github.com/ulope/pyformat.info/tree/v2) of the site that does away with the test based examples and will use [Lektor](https://getlektor.com) instead.

**Comment by ulope** [comment]
Thanks!

**Pull request merged**